ghcup-0.0.7.yaml

```diff
@@ -3785,12 +3785,110 @@ ghcupDownloads:
+          Linux_Fedora:
+            '>= 33':
+              dlHash: 21800b5faaf872d218766d9bf97b8d7f97df3cfae4b570dfd79d3327d9e3cdab
+              dlSubdir: ghc-9.6.3-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.6.3/ghc-9.6.3-x86_64-fedora33-linux.tar.xz
+            unknown_versioning: *ghc-963-64-centos7
+          Linux_Mint:
+            < 20: &ghc-963-64-ubuntu1804
+              dlHash: a499c05f554b6e6b7f04c81b376f1e569614e51864672c9226a696617bc6c640
+              dlSubdir: ghc-9.6.3-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.6.3/ghc-9.6.3-x86_64-ubuntu18_04-linux.tar.xz
+            '>= 20': &ghc-963-64-ubuntu2004
+              dlHash: d2018768b53ab2c9ab4d543d1e8d7c2b1fb78707b70c74c96ff1733e82f22b80
+              dlSubdir: ghc-9.6.3-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.6.3/ghc-9.6.3-x86_64-ubuntu20_04-linux.tar.xz
+          Linux_RedHat:
+            unknown_versioning: *ghc-963-64-centos7
+          Linux_Ubuntu:
+            ( >= 16 && < 19 ): *ghc-963-64-ubuntu1804
+            unknown_versioning: *ghc-963-64-ubuntu2004
+          Linux_UnknownLinux:
+            unknown_versioning:
+              dlHash: 56ed24e986749572f21c5249f43390fc7ae5078e3f296a33d693b4ad4b907fe1
+              dlSubdir: ghc-9.6.3-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.6.3/ghc-9.6.3-x86_64-rocky8-linux.tar.xz
+          Windows:
+            unknown_versioning:
+              dlHash: 0cc0e2d290a87c540268e62cc71f8eced2a24f621beeb524cce61bebf644f064
+              dlSubdir: ghc-9.6.3-x86_64-unknown-mingw32
+              dlUri: https://downloads.haskell.org/~ghc/9.6.3/ghc-9.6.3-x86_64-unknown-mingw32.tar.xz
+        A_ARM64:
+          Darwin:
+            unknown_versioning:
+              dlHash: e1cdf458926b2eaf52d2a8287d99a965040ff9051171f5c3b7467049cf0eb213
+              dlSubdir: ghc-9.6.3-aarch64-apple-darwin
+              dlUri: https://downloads.haskell.org/~ghc/9.6.3/ghc-9.6.3-aarch64-apple-darwin.tar.xz
+          Linux_UnknownLinux:
+            unknown_versioning:
+              dlHash: 03c389859319f09452081310fc13af7525063ea8930830ef76be2a14b312271e
+              dlSubdir: ghc-9.6.3-aarch64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.6.3/ghc-9.6.3-aarch64-deb10-linux.tar.xz
+      viChangeLog: https://downloads.haskell.org/~ghc/9.6.3/docs/users_guide/9.6.3-notes.html
+      viSourceDL:
+        dlHash: dfcde67b4aa550a0b8a1a9bb8105835dc999fad6397cce33d72fd55d21eb77f5
+        dlSubdir: ghc-9.6.3
+        dlUri: https://downloads.haskell.org/~ghc/9.6.3/ghc-9.6.3-src.tar.xz
+      viTags:
+      - Latest
+      - base-4.18.1.0
+      viTestDL:
+        dlHash: b0f92579288adf2320206627248d80d2abfe0e1b9ad2a22cf8f889bbe6806448
+        dlSubdir: ghc-9.6.3
+        dlUri: https://downloads.haskell.org/~ghc/9.6.3/ghc-9.6.3-src.tar.xz
```

Comment:
@chreekat this is wrong and broken... why did this happen?

Reply:
@hasufell I don't know what is wrong. What is it supposed to be?

@bgamari is there a bug in the metadata generation script?

@wz1000 fyi, in case this also impacts your ongoing releases.

Reply:
Ah, is it supposed to be `https://downloads.haskell.org/~ghc/9.6.3/ghc-9.6.3-testsuite.tar.xz`?

Reply:
Heh, yeah, that file matches the `dlHash`

Reply:
https://gitlab.haskell.org/ghc/ghc/-/issues/24268